cdap-ui/app/cdap/components/PipelineList/DeployedPipelineView/graphqlHelper.js

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright © 2019 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import { objectQuery } from 'services/helpers';
+import { GLOBALS } from 'services/global-constants';
+
+function getProgram(pipeline) {
+  const applicationDetail = objectQuery(pipeline, 'applicationDetail');
+
+  if (applicationDetail === null || applicationDetail === undefined) {
+    return [];
+  }
+
+  const programs = objectQuery(applicationDetail, 'programs');
+
+  if (programs === null || programs === undefined) {
+    return [];
+  }
+
+  const artifact = objectQuery(pipeline, 'artifact');
```

Comment:
fix the usage of objectQuery in this file

Reply:
you should still fix how you are getting the programs. You should see the implementation of objectQuery.

Reply:
Since objectQuery can return both undefined or null, do I still need to make changes to this function?

Reply:
you can just check if it's falsy `!artifact`

Reply:
Done